buildspec/packageTestVsix.yml

```diff
@@ -0,0 +1,23 @@
+version: 0.2
+
+env:
+    variables:
+        # needed or else webpack will cause it to run out of memory
+        NODE_OPTIONS: '--max-old-space-size=8192'
+
+phases:
+    install:
+        runtime-versions:
+            nodejs: 10
+
+    build:
+        commands:
+            # --unsafe-perm is needed because we run as root
+            - npm install --unsafe-perm
```

Comment:
Our other jobs have this as a prebuild step, should we move to that?

Reply:
it doesn't matter either way, this is a copy of our internal packaging script

Reply:
Works for me